pymatviz/ptable.py

```diff
@@ -785,9 +1187,6 @@ def ptable_hists(
     log: bool = False,
     anno_kwds: dict[str, Any] | None = None,
     on_empty: Literal["show", "hide"] = "hide",
-    color_elem_types: Literal["symbol", "background", "both", False]
-    | dict[str, str] = "background",
-    elem_type_legend: bool | dict[str, Any] = True,
     **kwargs: Any,
```

Comment:
maybe a misunderstanding. [when i said](https://github.com/janosh/pymatviz/pull/131#issuecomment-2058143028)

> i think we don't need `add_element_type_legend`. changing the background color of tiles definitely doesn't make sense. the color of the element symbol maybe but i think even that becomes visually too noisy

i was referring to the `ptable_splits` function. i think for `ptable_hists` these arguments make more sense, probably enough to keep them.

Reply:
Sure! I would revert it soon.

Reply:
Reverted.